bin/nodenv-each

```diff
@@ -2,32 +2,50 @@
 #
 # Summary: Execute a command for each Node version
 #
-# Usage: nodenv each [-v] <command> [arg1 arg2...]
+# Usage: nodenv each [-sv] <command> [arg1 arg2...]
 #
 # Executes a command for each Node version by setting NODENV_VERSION.
 # Failures are collected and reported at the end.
 #
+#    -s  Include the system `node`.
```

Comment:
I'm going to find a different short flag for this, besides `-s` because I'd like to keep s reserved for "silent".

I'm thinking either `-a` for "all" or `-g`, based on flags to other *nix utilities. I'm leaning towards: `--system, -g`